### PR TITLE
phoebus local value initialization

### PR DESCRIPTION
**Pull request by talwrii**

### CODE REVIEW

File: src/components/ConnectionWrapper/connectionWrapper.tsx [hunk 1]
```diff
@@ -7,22 +7,32 @@ import { VType } from "../../vtypes/vtypes";
...
+export interface PvErrorProps extends PvProps {
+  error: boolean;
+}
+
+
 function pvStateSelector(
   pvName: string,
   state: CsState
-): [boolean, boolean, VType?] {
+): [string, string | undefined, boolean, boolean, VType?] {
```
Comment (willrogers) [inline src/components/ConnectionWrapper/connectionWrapper.tsx]: `string | undefined` is `string?`

**Comment by willrogers** [comment]
I don't think you committed the `processVariable.tsx` file.

**Comment by willrogers** [comment]
I just tested on Phoebus and it seems that the first local PV to be initialised wins. That appears to be the first one in the XML file, but I'm not certain that that's deterministic.

**Comment by talwrii** [comment]
Okay this should be done.

File: src/connection/sim.ts [hunk 2]
```diff
@@ -356,81 +352,153 @@ export class SimulatorPlugin implements Connection {
...
-    } else if (parts[0] === "sim://enum") {
+      initial = undefined;
+    } else if (nameInfo.protocol === "sim://enum") {
       cls = SimEnumPv;
-    } else if (pvName.startsWith("enum://")) {
-      cls = EnumPv;
-    } else if (parts[0] === "sim://random") {
+      initial = undefined;
+    } else if (nameInfo.protocol === "enum://") {
+      return { simulator: undefined, initialValue: undefined };
```
Comment (willrogers) [inline src/connection/sim.ts]: We don't need this special case for `enum://`, just use the default.

**Pull request merged**